docs/template_plugin/src/CMakeLists.txt

```diff
@@ -30,7 +37,8 @@ target_link_libraries(${TARGET_NAME} PRIVATE
 
 set_target_properties(${TARGET_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
 
-# ATTENTION: uncomment to register a plugin in the plugins.xml file
-# ie_register_plugins(MAIN_TARGET ${TARGET_NAME}
-#                     POSSIBLE_PLUGINS ${TARGET_NAME})
+if (ENABLE_TEMPLATE_REGISTRATION)
+    # Update the plugins.xml file
+    ov_register_plugins(MAIN_TARGET ${TARGET_NAME})
```

Comment:
is it not enough just to put current `ov_register_plugins` under `if` without introducing the `SKIP_REGISTRATION` option for `ov_add_plugin`?